chains/solana/contracts/programs/fee-quoter/src/instructions/v1/messages.rs

```diff
@@ -116,7 +116,7 @@ pub fn process_extra_args(
             };
 
             let mut data = &extra_args[4..];
-            parse_and_validate_evm_extra_args(tag.try_into().unwrap(), &mut data)
+            parse_and_validate_generic_extra_args(tag.try_into().unwrap(), &mut data)
```

Comment:
should this be under `CHAIN_FAMILY_SELECTOR_EVM`? Or should it be the default option and fail in case the parsing does?
If we need to add any other chain that's not EVM but it's compliant with the default extra args, then no change in code should be needed.

Reply:
Good call, that does sound a bit more flexible.